deeplearning4j/deeplearning4j-nn/pom.xml

```diff
@@ -77,7 +77,17 @@
             <artifactId>nd4j-common</artifactId>
             <version>${nd4j.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.nd4j</groupId>
+            <artifactId>nd4j-tensorflow</artifactId>
+            <version>${nd4j.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
We can remove the test scope nd4j-tensorflow dependency from deeplearning4j-nn, we only need it in deeplearning4j-modelimport